h2o-core/src/main/java/hex/CreateFrame.java

```diff
@@ -43,7 +43,8 @@
   public Job<Frame> execImpl() {
     if (seed == -1) seed = new Random().nextLong();
     if (seed_for_column_types == -1) seed_for_column_types = seed;
-    if (integer_fraction + binary_fraction + categorical_fraction + time_fraction + string_fraction > 1) throw new IllegalArgumentException("Integer, binary, categorical, time and string fractions must add up to <= 1.");
+    if (integer_fraction + binary_fraction + categorical_fraction + time_fraction + string_fraction > 1.00000001)
```

Comment:
Can check a diff from 1 here and add a tolerance in scientific notation, but thats just nit picking.